### PR TITLE
Lvphpiii 28

### PR DESCRIPTION
**Pull request by kovalc0mrade**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPHPIII-XXX)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVPHPIII-XXX)

## Code reviewers

- [ ] @ecconomix
- [ ] @kovalc0mrade
- [ ] @NFalinska
- [ ] @NikitaKompanetc
- [ ] @nykolaiko
- [ ] @Roksoliana
- [ ] @topvova

### Second Level Review

- [ ] @masJIan
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST

- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/Repositories/QueueRepository.php [hunk 1]
```diff
@@ -37,4 +50,36 @@ public function getSliderQueueData()
         }
         return $result;
```
Comment (kovalc0mrade) [inline app/Repositories/QueueRepository.php]: слайдер не мій

File: app/Http/Controllers/v1/FileController.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Controllers\v1;
+
+use App\Repositories\FileRepository;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+class FileController extends Controller
```
Comment (masJIan) [inline app/Http/Controllers/v1/FileController.php]: Annotations

File: app/Http/Controllers/v1/HelpersController.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+namespace App\Http\Controllers\v1;
+
+use App\Repositories\CityRepository;
+use App\Repositories\CountryRepository;
+use App\Repositories\PassportRepository;
+use App\Repositories\ProposalRepository;
+use App\Repositories\SexRepository;
+use App\Http\Controllers\Controller;
+
+class HelpersController extends Controller
```
Comment (masJIan) [inline app/Http/Controllers/v1/HelpersController.php]: Annotations

File: app/Http/Controllers/v1/UserController.php [hunk 4]
```diff
@@ -3,16 +3,94 @@
...
+
+        $queueId = $queueRepository->createQueue();
+
+        if (!$requestRepository->createRequest(
+            $request->all(),
+            $queueId,
+            $userId
+        )) {
+            return response()->json([
+                'error' => 'Can\'t create request'
```
Comment (masJIan) [inline app/Http/Controllers/v1/UserController.php]: Hardcoded values =)

**Pull request merged**